ios/FluentUI/Tooltip/TooltipView.swift

```diff
@@ -136,12 +153,127 @@ class TooltipView: UIView {
+                                                                   paddingHorizontal: 0,
+                                                                   labelFont: titleLabelFont)
+            titleLabel.frame.size = preferredTitleSize
+            messageLabel.frame.origin = CGPoint(x: 0, y: titleLabel.frame.height + spacingVertical)
+        }
+
+        // Shadow
+        let ambientShadow = CALayer()
+        ambientShadow.frame = bounds
+        ambientShadow.shadowPath = UIBezierPath(roundedRect: bounds, cornerRadius: backgroundCornerRadius).cgPath
+        ambientShadow.shadowColor = UIColor(dynamicColor: shadowInfo.colorOne).cgColor
+        ambientShadow.shadowOpacity = 1
+        ambientShadow.shadowOffset = CGSize(width: shadowInfo.xOne, height: shadowInfo.yOne)
+        ambientShadow.shadowRadius = shadowInfo.blurOne
+        layer.insertSublayer(ambientShadow, at: 0)
+
+        let perimeterShadow = CALayer()
+        perimeterShadow.frame = bounds
+        perimeterShadow.shadowPath = UIBezierPath(roundedRect: bounds, cornerRadius: backgroundCornerRadius).cgPath
+        perimeterShadow.shadowColor = UIColor(dynamicColor: shadowInfo.colorTwo).cgColor
+        perimeterShadow.shadowOpacity = 1
+        perimeterShadow.shadowOffset = CGSize(width: shadowInfo.xTwo, height: shadowInfo.yTwo)
+        perimeterShadow.shadowRadius = shadowInfo.blurTwo
+        layer.insertSublayer(perimeterShadow, at: 0)
+    }
+
+    func updateAppearance(tooltipColor: UIColor,
+                          textColor: UIColor,
+                          shadowInfo: ShadowInfo,
+                          backgroundCornerRadius: CGFloat) {
+        // Update colors
+        backgroundView.backgroundColor = tooltipColor
+        messageLabel.textColor = textColor
+        titleLabel?.textColor = textColor
+
+        // Update shadows
+        layer.sublayers?.removeAll()
+        let ambientShadow = CALayer()
+        ambientShadow.frame = bounds
+        ambientShadow.shadowPath = UIBezierPath(roundedRect: bounds, cornerRadius: backgroundCornerRadius).cgPath
+        ambientShadow.shadowColor = UIColor(dynamicColor: shadowInfo.colorOne).cgColor
+        ambientShadow.shadowOpacity = 1
+        ambientShadow.shadowOffset = CGSize(width: shadowInfo.xOne, height: shadowInfo.yOne)
+        ambientShadow.shadowRadius = shadowInfo.blurOne
+        layer.insertSublayer(ambientShadow, at: 0)
+
+        let perimeterShadow = CALayer()
+        perimeterShadow.frame = bounds
+        perimeterShadow.shadowPath = UIBezierPath(roundedRect: bounds, cornerRadius: backgroundCornerRadius).cgPath
+        perimeterShadow.shadowColor = UIColor(dynamicColor: shadowInfo.colorTwo).cgColor
+        perimeterShadow.shadowOpacity = 1
+        perimeterShadow.shadowOffset = CGSize(width: shadowInfo.xTwo, height: shadowInfo.yTwo)
+        perimeterShadow.shadowRadius = shadowInfo.blurTwo
+        layer.insertSublayer(perimeterShadow, at: 0)
```

Comment:
Seems like you are adding both shadow layers on the same index.

Reply:
it just inserts it at that place! so the first one gets moved to index 1